README.md

```diff
@@ -1 +1,239 @@
-# mathematician
\ No newline at end of file
+# Leaderboard
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📐 [Mathematician:] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+This is the first project of the Math Magicians application.
+
+**[Leaderboard: setup]** It is leaderboard Api using Js.
```

Comment:
- [ ] I suggest you remove this information from your README as it does not relate to this project